dts/arm/nuvoton/npcx7m6fb.dtsi

```diff
@@ -26,6 +28,18 @@
 		compatible = "mmio-sram";
 		reg = <0x200C0000 0x10000>;
 	};
+
+	soc {
+		/*********************** Clock controller *********************/
```

Comment:
I think you should drop the **** line. We can see it is the clock controler :-)

Reply:
OK